package.json

```diff
@@ -10,7 +10,7 @@
     "prod": "cross-env NODE_ENV=production pm2-runtime start ./build/server.js",
     "start": "cross-env pm2-runtime start ./build/server.js --watch",
     "setup": "tsx setup.ts",
-    "test": "vitest run --no-threads --coverage",
+    "test": "vitest run --pool=threads --coverage",
```

Comment:
does this parallelize the tests? if it does then one thing to keep in mind, mutations tests can be isolated since they should work on their own database objects, but query tests cannot be parallelized because of data mismatch

the assumption is based on the fact that only a single database is used while testing so test data for each test is contained within a single database and could affect data comparision logic within those tests